app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.